### ISSUE TITLE
Fail immediately if test crashes

### ISSUE THREAD

**Issue by tcies** [ethz-asl/continuous_integration#45]
Would it be possible to fail immediately if a test crashes (crash != fail, missing xml)? This should save unnecessary jenkins compute and would have the advantage that the segfault stack trace could be easily found by inspecting the tail of the console output.

**Comment by simonlynen** [comment]
Yes you could do a bit of parsing here: https://github.com/ethz-asl/continuous_integration/blob/master/catkin_make_repo#L35

**Comment by simonlynen** [comment]
Partially adr https://github.com/ethz-asl/continuous_integration/pull/46
